source/includes/administration/_custom_ui_settings.md

```diff
@@ -0,0 +1,246 @@
+### Custom UI Settings
+
+<!-------------------- FIND CUSTOM UI SETTINGS -------------------->
+
+#### Find custom ui settings associated to an organization
+
+`GET /reseller/settings/custom_ui/find?organizationId=:id`
+
+Retrieve the custom ui settings associated to an organization. If the `organizationId` is omitted, the authenticated user's organization will be used.
```

Comment:
To be consistent with rest of docs we can capitalize **UI** here

Reply:
e1fe79e8e800d38760fbeeac4d2ed4a17474aa49